Arbeidskrav2.Rmd

```diff
@@ -3,15 +3,20 @@ title: "Arbeidskrav 2, inference"
 author: "Magnus Kleiven"
 date: "20 10 2020"
 output: html_document
+editor_options: 
+  chunk_output_type: console
 ---
 
-Innledning og hypotese:
-I denne oppgaven skal vi se på hvorvidt en treningsintervensjon påvirker totalt RNA innhold i muskelen, og om denne påvirkningen er forskjellig fra personer som defineres som "high" eller "low" responders på treningsintervensjonen. 
+# Innledning og hypotese:
+I denne oppgaven skal vi se på hvorvidt en treningsintervensjon påvirker totalt RNA innhold i muskelen, og om denne påvirkningen er forskjellig fra personer som defineres som "high" eller "low" responders på treningsintervensjonen.
+<!-- In a final report (mappeeksamen), maybe add some information on why you expect this result -->

```

Comment:
`<!--` and `-->` is the start and end of html comment, see soem of the commenst in the Rmd file